tests/cases/fourslash/renameDestructuringAssignmentInForOf2.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference path='fourslash.ts' />
+
+////interface I {
+////    property1: number;

```

Comment:
Don't think you want to combine that.. Just because you happen to rename the name of variable (say so it contains all types of properties at various instances) doesn't mean you want to rename the properties. So current behavior is what you want there